src/components/analytics/GoogleAnalyticsProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import ReactGA from 'react-ga4'
+import { GaOptions, InitOptions, UaEventOptions } from 'react-ga4/types/ga4'
+
+/**
+ * Google Analytics Provider containing all methods used throughout app to log events to Google Analytics.
+ */
+export default class GoogleAnalyticsProvider {
```

Comment:
We did something similar on our side, didn't we?

Reply:
Yes, we have our own implementation